onnxconverter_common/data_types.py

```diff
@@ -248,9 +248,9 @@ def find_type_conversion(source_type, target_type):
     """
     Find the operator name for converting source_type into target_type
     """
-    if type(source_type) == type(target_type):
+    if isinstance(source_type, target_type):
```

Comment:
Convert from coreml calls this with target_type being FloatTensorType(shape=[]), which is not a type, resulting in 
`TypeError: isinstance() arg 2 must be a type, a tuple of types, or a union`

Writing here because the old version would have worked

Python 3.10.13
onnxconverter_common.__version__ 1.14.0
onnxmltools.__version__ 1.12.0
onnx.__version__ 1.15.0
coremltools.__version__ 7.1